src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -146,7 +138,9 @@ The .NET Foundation licenses this file to you under the MIT license.
       <IlcToolsPath Condition="'$(IlcToolsPath)' == ''">$(IlcHostPackagePath)\tools\</IlcToolsPath>
       <IlcSdkPath Condition="'$(IlcSdkPath)' == ''">$(RuntimePackagePath)\sdk\</IlcSdkPath>
       <IlcFrameworkPath Condition="'$(IlcFrameworkPath)' == ''">$(RuntimePackagePath)\framework\</IlcFrameworkPath>
-      <IlcFrameworkNativePath Condition="'$(IlcFrameworkNativePath)' == ''">$(RuntimePackagePath)\framework\</IlcFrameworkNativePath>
+      <IlcFrameworkNativePath Condition="'$(RuntimePackagePath)' != '' and '$(IlcFrameworkNativePath)' == ''">$(RuntimePackagePath)\framework\</IlcFrameworkNativePath>
+      <IlcFrameworkNativePath Condition="'$(IlcFrameworkPath)' != '' and '$(IlcFrameworkNativePath)' == ''">$(IlcFrameworkPath)</IlcFrameworkNativePath>
```

Comment:
In tests, `IlcFrameworkPath` is set elsewhere but `RuntimePackagePath` never is. My understanding is in this case we want `IlcFrameworkNativePath` to match `IlcFrameworkPath`.

I'm not sure what we should do in a scenario where `IlcFrameworkPath` is set before `SetupProperties` to something besides `$(RuntimePackagePath)\framework\`, but `RuntimePackagePath` is also set before `SetupProperties`. Would we want `IlcFrameworkNativePath` to be set to `$(RuntimePackagePath)\framework\`, or should it always be set to `IlcFrameworkPath`. And if it's the latter, do we need separate properties? @MichalStrehovsky @LakshanF Any thoughts?

Reply:
In src/tests/build.proj, we're setting up a bunch of properties IlcFrameworkPath, but we don't set IlcFrameworkNativePath. If we set it there would it fix the problem/avoid having to add the dummy targets?

Reply:
Yeah, that should solve the issue of how to set IlcFrameworkNativePath.

I'm not sure we actually need the dummy targets in the first place based on https://learn.microsoft.com/en-us/visualstudio/msbuild/target-build-order?view=vs-2022. I think `SetupProperties` wasn't running because the `IlcDynamicBuildDependencies` property used in the `DependsOnTargets` wasn't passed from src/tests/build.proj.